### PR TITLE
More complete wadl for eidaws/wfcatalog

### PR DESCRIPTION
**Pull request by kaestli**
The wadl-s currently installed with the EIDA end nodes lack descriptions of the data subselection parameters with comparison suffixes. @jbienkowski just released a new, more complete wadl. Propose to use this also for the federated wfcatalog service.

### CODE REVIEW

**Comment by damb** [comment]
@kaestli, I reassigned the base branch to `EIDA:next` (in accordance with [git-flow](https://nvie.com/posts/a-successful-git-branching-model/)).

File: eidangservices/federator/share/wfcatalog.wadl [hunk 1]
```diff
@@ -54,62 +54,326 @@
...
+            <param name="glitches_lt" style="query" type="xsd:float"/>
+            <param name="glitches_le" style="query" type="xsd:float"/>
             <param name="missing_padded_data" style="query" type="xsd:float"/>
+            <param name="missing_padded_data_eq" style="query" type="xsd:float"/>
+            <param name="missing_padded_data_ne" style="query" type="xsd:float"/>
+            <param name="missing_padded_data_gt" style="query" type="xsd:float"/>
+            <param name="missing_padded_data_ge" style="query" type="xsd:float"/>
+            <param name="missing_padded_data_lt" style="query" type="xsd:float"/>
+            <param name="missing_padded_data_le" style="query" type="xsd:float"/>
             <param name="telemtry_sync_error" style="query" type="xsd:float"/>
```
Comment (damb) [inline eidangservices/federator/share/wfcatalog.wadl]: Could you please fix the typo here, too (see also PR https://github.com/EIDA/wfcatalog/pull/15)? Thx.

Comment (jbienkowski) [reply eidangservices/federator/share/wfcatalog.wadl]: Done, thanks!

Comment (damb) [reply eidangservices/federator/share/wfcatalog.wadl]: Thanks guys.

**Pull request merged**